test/test-fs.c

```diff
@@ -37,6 +37,9 @@
 # ifndef ERROR_SYMLINK_NOT_SUPPORTED
 #  define ERROR_SYMLINK_NOT_SUPPORTED 1464
 # endif
+# ifndef S_IFIFO
+#  define S_IFIFO _S_IFIFO
```

Comment:
It looked like this is already added in the public header, so why is it needed here too?

Reply:
The one in uv/win.h is guarded by this:
```c
#if defined(_CRT_INTERNAL_NONSTDC_NAMES) && _CRT_INTERNAL_NONSTDC_NAMES && !defined(S_IFIFO)
```
Which VS 2015 doesn't seem to define. I think it was added in 2017 but don't quote me on that.

Maybe the guard is wrong but c17bd99f1 appears to have added it quite intentionally. You reviewed the PR so I assume it's okay?

Reply:
I assumed the condition was acceptable because it passed CI on VS2017+, and didn't check on an older version. But it should not matter, since it is only a new feature.